ITR/data/data_warehouse.py

```diff
@@ -36,8 +36,13 @@ def __init__(self, company_data: CompanyDataProvider,
         self.temp_config = tempscore_config
         self.column_config = column_config
         self.company_data = company_data
-        self.company_data._calculate_target_projections(benchmark_projected_production)
-        
+        self.company_data._calculate_target_projections(benchmark_projected_production,
+                                                        benchmarks_projected_ei.primary_scope)
+
+        # If benchmark's primary scope is S3 - skip shifting data into S1S2
+        if benchmarks_projected_ei.primary_scope == EScope.S3:
```

Comment:
Maybe the thing to do here is create some nifty lambda functions that can be used to steer S3 data to the correct bucket according to the benchmark, and not try to track the conditional logic everywhere.  Just have the right sent of lambdas for the right actions in the right places.

Reply:
Lambdas could be here, yes. In general, any task may be done in multiple ways. This function, and it's scope shift logic, is already written on conditional checks. When I add functionality, I try to avoid unnecessary refactoring.
If you think switching this function to lambdas is beneficial - could we create a separate task for it, and somebody will tackle it later? On the other hand, if lambdas mean more work for no benefit - maybe we should keep the logic conditional

Reply:
This becomes an implementation detail of the larger comment above about not prematurely or wrongly combining scopes while computing EI.

Reply:
In an alternate world, one could ask and answer the question "what would the company's temperature score be if we looked only at S3 emissions?"  And we can do the calculation of cumulative projected S3 vs. total budgeted S3 to come up with overshoot/undershoot multiplied by TCRE to get ITR for scope 3.  And it would give information that when looking only at S3, these companies overshoot and those companies undershoot.  And it will shake out bugs where scopes are wrongly converted from one scope to another.

But...because the benchmark budget is ultimately a global budget that has been carefully calibrated and divided among sectors AND scopes, there's no meaning to an S3-only score.  The only meaningful final score is the combined scopes of the company vs. the combined scopes of the benchmark.

I don't think it could be useful to show how the under/over on the individual scopes could help investors who want to prioritize S1S2 over S3 (or vice-versa), and so I don't want to dissuade from that.  But want to be clear what the various components mean within the process (i.e., scope emissions, scope emissions intensity, values vs. budget, etc).

Reply:
> there's no meaning to an S3-only score

my whole task is to calculate S3-only score. The [GitHub issue  146](https://github.com/os-climate/ITR/issues/146) is about S3-only score. You added an IE bemchmark entry, called "OECM (Scope 3) 1.5 degC".

This tells me, that there is value in S3-only score.

If you want to fix S1S2 folding in this PR, i highly recommend not to mix this work into the work, intended to be solved by this PR

Reply:
In slack and calls, we discussed, and AFAIU agreed, that separate S3-only calculation is helpful, as a single separate step of continuous development of the S1S2S3 calculation in ITR tool.
Therefore, I close this comment thread as resolved